src/Microsoft.AspNet.Razor/Parser/SyntaxTree/BlockBuilder.cs

```diff
@@ -21,7 +21,7 @@ public BlockBuilder(Block original)
         }
 
         public BlockType? Type { get; set; }
-        public IList<SyntaxTreeNode> Children { get; private set; }
+        public List<SyntaxTreeNode> Children { get; private set; }
```

Comment:
`Block` doesn't need to change it but this is a builder type. There'd be no way to add `Children` to `BlockBuilder`s unless I added an additional method to add children (not worth when its true type has the capability already).